app/src/components/Main.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Container, Navbar, Nav, Dropdown, NavItem, Card } from 'react-bootstrap';
+
+import * as styles from './Header.module.scss';
+import NavLink from 'react-bootstrap/NavLink';
+
+const BreadCrumb: React.FC<{crumbs: string[]}> = ({ crumbs }) => {
+  return <div className={styles.breadcrumb}>
+    {crumbs.map((value: string, index: number) => {
+      return <>
+        <a href="" className={styles.link}>{value}</a>
+        {index === (crumbs.length - 1) ? null : <span className={styles.separator}></span>}
+      </>;
+    })}
+  </div>;
+};
+
+interface User {
+  img: string;
+  name: string;
+  role: string;
+}
+
+const UserCard: React.FC<{user: User}> = ({ user }) => {
+  return <Card className={styles.testimonialCard}>
+    <div className={styles.blueGradient + ' ' + styles.cardUp}>
+    </div>
+    <div className={'mx-auto white ' + styles.avatar}>
+      <img src={user.img} className="rounded-circle img-fluid"/>
+    </div>
+    <Card.Body>
+      <h4 className="font-weight-bold mb-4">{user.name} <small>{user.role}</small></h4>
+      <hr/>
+    </Card.Body>
+  </Card>;
+};
+
+const UserDropdown: React.FC<{user: User}> = ({ user }) => {
+  return <Dropdown as={NavItem} alignRight>
+    <Dropdown.Toggle as={NavLink} id="nav-dropdown">
+      <img src={user.img} className="rounded-circle z-depth-0" style={{ 'width': '3rem' }}/>
+    </Dropdown.Toggle>
+    <Dropdown.Menu className={styles.avatarDropdown}>
+      <UserCard user={user}/>
+    </Dropdown.Menu>
+  </Dropdown>;
+};
+
+const Header: React.FC<{page: string; parents: string[]}> = ({ page, parents }) => {
+  return (
+    <Navbar collapseOnSelect expand="lg">
```

Comment:
atm all the links in the navbar go to the same page. 
e.g. in 
![image](https://user-images.githubusercontent.com/3920879/68776180-91555000-0627-11ea-99d2-5d6d5f6aac1d.png)
both 'Management' and 'Artifact Requests' link to 'Artifact Requests'. Is that the desired behaviour?

Reply:
Yeah I haven't actually linked them to anything. Mainly because I think it's useful as a UI context but you'll use the navigation bar practically.